e2e-tests/tests/add-static-page.spec.ts

```diff
@@ -10,7 +10,12 @@ const expectedPage = fs.readFileSync(
 studioTest("can add a static page", async ({ page, studioPage }) => {
   const pageInTree = page.getByText("NewStaticPage");
   await expect(pageInTree).toHaveCount(0);
-  await studioPage.addStaticPage("NewStaticPage", "index.html");
+
+  const addPageSection = studioPage.addPageSection;
+  await addPageSection.selectPageType(false, true);
+  await addPageSection.enterBasicPageData("NewStaticPage", "index.html", true);
+  await addPageSection.selectLayout();
```

Comment:
Although we aren't selecting a layout, do we want to keep the screenshot for the layout selection modal for visual regression purposes?

Reply:
We have a screenshot already! It's part of the `add-page-with-layout.spec.ts` test. When I cut that test over to use the new Section, I will make sure to keep the layout screenshot.